src/Backend/SgateShopgatePlugin/Bootstrap.php

```diff
@@ -1009,6 +1009,23 @@ public function onBackendAfterSaveOrder(Enlight_Event_EventArgs $args)
             case "savePosition":
                 $orderId = $args["subject"]->Request()->getParam("orderId");
                 break;
+            case "batchProcess":
+                $orders = $args["subject"]->Request()->getParam('orders', array(0 => $args["subject"]->Request()->getParams()));
+
+                if (!empty($orders)) {
+                    foreach ($orders as $key => $data) {	
+                        if (empty($data) || empty($data['id'])) {
+			    continue;
+                        }
+                        
+			$orderId = $data['id'];
+                        if ($orderId) {
+                            self::cancelOrder($orderId);
+                            self::confirmOrderShipping($orderId);
+                        }
+                    }
+                }
+		break;
```

Comment:
This `break` statement should be replaced by a return statement.
Reason: with `brake`, the rest of the code would be executed and $orderId being undefined will cause a warning.